packages/react-components/src/components/chart/trendCursor/tests/useVisualizedDataStreams.spec.ts

```diff
@@ -9,7 +9,7 @@ it('converts empty query to empty data stream', async () => {
     result: { current },
```

Comment:
worth adding some tests for the unhide/ hide  and highlighting operations ?

Reply:
I agree. Going to cut a ticket for this because I really need to start on composition API for re:Invent and revisit it.